lib/index.js

```diff
@@ -10,10 +10,13 @@ var oauth2_client;
  *  @returns {Object} oauth2_client - the Google OAuth2 client
  */
 function create_oauth2_client () {
+  var google_client_id = (OPTIONS.google_client_id) ? OPTIONS.google_client_id : process.env.GOOGLE_CLIENT_ID;
```

Comment:
Yes, this gives the option of setting Google client ID / secret in either the options object or via the environment variable. The thinking here was that if someone wants to set the secret in options they could add google_client_id and google_client_secret for visibility in the actual options code, rather than relying on the environmental variable as not all environments will use environmental variables to pass configuration specific parameters. 

The way I coded this, it maintains the old way of setting Google secrets via the environmental variable but also allows you to use your own scheme in the options.